.github/workflows/build_msi.yml

```diff
@@ -21,14 +23,14 @@ jobs:
     name: 'Windows (x86) Installer'
     runs-on: windows-latest
     steps:
-    - uses: actions/checkout@v1
```

Comment:
Can you please revert these changes and write a separated PR?

Reply:
Done!

Reply:
And new PR: https://github.com/python/cpython/pull/20232